sdk/monitor/azure-monitor-opentelemetry-exporter/pom.xml

```diff
@@ -46,6 +46,11 @@
       <artifactId>azure-core</artifactId>
       <version>1.49.1</version> <!-- {x-version-update;com.azure:azure-core;dependency} -->
     </dependency>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-json</artifactId>
+      <version>1.1.0</version> <!-- {x-version-update;com.azure:azure-json;dependency} -->
+    </dependency>
```

Comment:
is jackson dependency automatically removed, or do we need something to drop that (to help with agent jar file size)

Reply:
it's a transient, i guess i can exclude it.. it was from azure.core. let me double check the dependency tree.

Reply:
it will be removed from azure-core once the migration is complete for the whole repo.